tests/components/huisbaasje/test_sensor.py

```diff
@@ -133,7 +133,7 @@ async def test_setup_entry(hass: HomeAssistant):
         )
         assert (
             energy_consumption_peak_today.attributes.get(ATTR_STATE_CLASS)
-            is STATE_CLASS_TOTAL_INCREASING
+            is SensorStateClass.TOTAL_INCREASING
```

Comment:
`is STATE_CLASS_TOTAL_INCREASING` was causing issues, so I had to change it to use `SensorStateClass`

I also took the opportunity to update all the corresponding lines to have the same methodology inside the file